src/modules/files.ts

```diff
@@ -113,4 +113,15 @@ export default class Files extends API {
     )
     return response.json()
   }
+
+  /**
+   * Get raw data for a File
+   *
+   * @param file File object
+   * @returns Raw data
+   */
+  async fileData(hash: string): Promise<string> {
+    const response = await this.sendGet(`file/${hash}`)
+    return response.text() as Promise<string>
```

Comment:
if that endpoint returns raw bytes, we can get that here as well... might be more useful if you're doing things like grabbing raw data to convert to objects or similar?

Reply:
We can get the response as an arrayBuffer or blob if we want: https://developer.mozilla.org/en-US/docs/Web/API/Response#Methods

Reply:
cool. that fetch lib suggested to use response.text() here so i just assumed to do it that but maybe they aren't mutually exclusive. i'm going to move this to a ticket, can you try to add? #